### PR TITLE
Release notes for edge-19.4.4

### PR DESCRIPTION
**Pull request by dadjeibaah**
Signed-off-by: Dennis Adjei-Baah <dennis@buoyant.io>

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+## edge-19.4.4
+* Proxy
+  * **Fixed** a connection starvation issue where TLS discovery detection on
+    on slow or idle connections could block all other connections from being
```
Comment (siggy) [inline CHANGES.md]: `on on`

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+    accepted on the inbound listener of the proxy
+* CLI
+  * **Fixed** `inject` to allow the `--disable-identity` flag to be used
+    without having to specify the `--ingore-cluster` flag
```
Comment (siggy) [inline CHANGES.md]: `--ignore-cluster`

Comment (ihcsim) [inline CHANGES.md]: Typo: `--ingore-cluster`

**Comment by scottcarol** [comment]
This was not from you, but I noticed that `## edge-19.4.1` comments are not 80 characters, might be worth changing for readability TIOLI

**Comment by dadjeibaah** [comment]
@scottcarol good catch! I think I'll address that in another PR.

**Pull request merged**